### PR TITLE
docs: brief notes on NPM/CommonJS/ES6 usage

### PR DESCRIPTION
**Pull request by dhdaines**
### Short description of changes:

Add basic documentation about using `wavesurfer.js` with NPM and CommonJS/ES6 modules 

### Breaking in the external API:

nothing

### Breaking changes in the internal API:

nothing

### Todos/Notes:

nothing

### Related Issues and other PRs:

https://github.com/katspaugh/wavesurfer.js/issues/1077
https://github.com/katspaugh/wavesurfer.js/issues/2427

### CODE REVIEW

File: plugins/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h1>Plugins</h1>
 
   <h2>How to configure a plugin</h2>
 
-  <p>To start using a plugin, you need to insert the plugin library into your HTML page, alongside <code>wavesurfur.js</code>, like this:</p>
+  <p>To start using a plugin, you need to insert the plugin library into your HTML page, alongside <code>wavesurfer.js</code>, like this:</p>
```
Comment (thijstriemstra) [inline plugins/index.html:9]: good catch!

File: plugins/index.html [hunk 2]
```diff
@@ -17,6 +17,19 @@ <h2>How to configure a plugin</h2>
     ]
 });</code></pre>
 
+  <p>When using NPM, the plugins are installed
```
Comment (thijstriemstra) [inline plugins/index.html:20]: this sentence isn't flowing correctly..

File: plugins/index.html [hunk 3]
```diff
@@ -17,6 +17,19 @@ <h2>How to configure a plugin</h2>
...
+  in <code>node_modules</code> with the package but must be imported or required
+  locally in order to use them:
+    <pre><code class="javascript">
+# ES6: import RegionsPlugin from "wavesurfer.js/dist/plugin/wavesurfer.regions.min.js";
```
Comment (thijstriemstra) [inline plugins/index.html:24]: a hastag isn't a valid comment identifier in javascript, use // instead

File: plugins/index.html [hunk 4]
```diff
@@ -17,6 +17,19 @@ <h2>How to configure a plugin</h2>
...
+const RegionsPlugin = require("wavesurfer.js/dist/plugin/wavesurfer.regions.min.js");
```
Comment (thijstriemstra) [inline plugins/index.html:25]: use var instead of const for max compat

File: plugins/index.html (hunk 2 above)
Comment (dhdaines) [reply plugins/index.html]: I think I fixed it in c3a2edb...

File: plugins/index.html (hunk 3 above)
Comment (dhdaines) [reply plugins/index.html]: ah! good idea, sorry!

**Review by thijstriemstra — APPROVED** [review]
thanks!

**Pull request merged**